fw/http_parser.c

```diff
@@ -1873,6 +1873,8 @@ __req_parse_content_type(TfwHttpMsg *hm, unsigned char *data, size_t len)
 	__FSM_STATE(I_EoL) {
 		__FSM_I_MATCH_MOVE_fixup(ctext_vchar, I_EoL, 0);
 		if (IS_CRLF(*(p + __fsm_sz))) {
+			/* __TFW_HTTP_PARSE_SPECHDR_VAL is told to not fixup it */
+			__msg_hdr_chunk_fixup(p, __fsm_sz);
```

Comment:
Well, this does fix https://github.com/tempesta-tech/tempesta/issues/1296#issuecomment-1008446777 . However, the fix looks like a workaround for `__TFW_HTTP_PARSE_SPECHDR_VAL(..., 0)` and we can also fix the issue simply with calling the function from `TFW_HTTP_PARSE_SPECHDR_VAL()`. However, I see that there are a lot of headers with values parsed with `__TFW_HTTP_PARSE_SPECHDR_VAL(..., 0)` instead of `TFW_HTTP_PARSE_SPECHDR_VAL()` - do we have problems with them as well? Seems not. So need to figure out why some of the headers are parsed with the first macro and others with the second one. Please write a good comments at least for the macros when each of them must be used. If the problem should be fixed exactly in the was as in the patch, then please write a good comment why - at this moment it's not clear.